### PR TITLE
Adding support for v3.4

### PR DESCRIPTION
**Pull request by DamianAmbrozkiewicz**

### CODE REVIEW

File: livechat/agent/rtm/client.py [hunk 1]
```diff
@@ -967,4 +970,10 @@ def list_agents_for_transfer(self,
 
 
 class AgentRTM33(AgentRTMInterface):
-    ''' AgentRTM version 3.3 class. '''
+    ''' Agent RTM version 3.3 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/agent/rtm/client.py:974]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/agent/rtm/client.py [hunk 2]
```diff
@@ -967,4 +970,10 @@ def list_agents_for_transfer(self,
...
+
+
+class AgentRTM34(AgentRTMInterface):
+    ''' Agent RTM version 3.4 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/agent/rtm/client.py:979]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/agent/web/client.py [hunk 3]
```diff
@@ -957,4 +958,10 @@ def list_agents_for_transfer(self,
 
 
 class AgentWeb33(AgentWebInterface):
-    ''' Customer API version 3.3 class. '''
+    ''' Agent API version 3.3 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/agent/web/client.py:962]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/agent/web/client.py [hunk 4]
```diff
@@ -957,4 +958,10 @@ def list_agents_for_transfer(self,
...
+
+
+class AgentWeb34(AgentWebInterface):
+    ''' Agent API version 3.4 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/agent/web/client.py:967]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/configuration/client.py [hunk 5]
```diff
@@ -977,3 +978,9 @@ def get_license_webhooks_state(self,
 
 class ConfigurationApi33(ConfigurationApiInterface):
     ''' Configuration API client in version 3.3 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/configuration/client.py:981]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/configuration/client.py [hunk 6]
```diff
@@ -977,3 +978,9 @@ def get_license_webhooks_state(self,
...
+
+
+class ConfigurationApi34(ConfigurationApiInterface):
+    ''' Configuration API client in version 3.4 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/configuration/client.py:986]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/customer/rtm/client.py [hunk 7]
```diff
@@ -623,5 +624,10 @@ def update_event_properties(self,
 
 
 class CustomerRTM33(CustomerRTMInterface):
-    ''' CustomerRTM version 3.3 class. '''
+    ''' Customer RTM version 3.3 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/customer/rtm/client.py:628]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/customer/rtm/client.py [hunk 8]
```diff
@@ -623,5 +624,10 @@ def update_event_properties(self,
...
+
+
+class CustomerRTM34(CustomerRTMInterface):
+    ''' Customer RTM version 3.4 class. '''
     pass
```
Comment (marcindebski) [inline livechat/customer/rtm/client.py:633]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/customer/web/client.py [hunk 9]
```diff
@@ -835,3 +836,9 @@ def cancel_greeting(self,
 
 class CustomerWeb33(CustomerWebInterface):
     ''' Customer API version 3.3 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/customer/web/client.py:839]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

File: livechat/customer/web/client.py [hunk 10]
```diff
@@ -835,3 +836,9 @@ def cancel_greeting(self,
...
+
+
+class CustomerWeb34(CustomerWebInterface):
+    ''' Customer API version 3.4 class. '''
+    pass
```
Comment (marcindebski) [inline livechat/customer/web/client.py:844]: pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

**Review by marcindebski — COMMENTED** [review]
We need to decide in order to have a common approach (`pass` is not necessary (https://www.python.org/dev/peps/pep-0257/))

File: livechat/customer/web/client.py (hunk 10 above)
Comment (DamianAmbrozkiewicz) [reply livechat/customer/web/client.py:844]: 👍

File: livechat/customer/web/client.py (hunk 9 above)
Comment (DamianAmbrozkiewicz) [reply livechat/customer/web/client.py:839]: 👍

File: livechat/customer/rtm/client.py (hunk 8 above)
Comment (DamianAmbrozkiewicz) [reply livechat/customer/rtm/client.py:633]: 👍

File: livechat/customer/rtm/client.py (hunk 7 above)
Comment (DamianAmbrozkiewicz) [reply livechat/customer/rtm/client.py:628]: 👍

File: livechat/configuration/client.py (hunk 5 above)
Comment (DamianAmbrozkiewicz) [reply livechat/configuration/client.py:981]: 👍

File: livechat/agent/web/client.py (hunk 4 above)
Comment (DamianAmbrozkiewicz) [reply livechat/agent/web/client.py:967]: 👍

File: livechat/agent/web/client.py (hunk 3 above)
Comment (DamianAmbrozkiewicz) [reply livechat/agent/web/client.py:962]: 👍

File: livechat/agent/rtm/client.py (hunk 2 above)
Comment (DamianAmbrozkiewicz) [reply livechat/agent/rtm/client.py:979]: 👍

File: livechat/agent/rtm/client.py (hunk 1 above)
Comment (DamianAmbrozkiewicz) [reply livechat/agent/rtm/client.py:974]: 👍

**Review by marcindebski — APPROVED** [review]

**Pull request merged**